src/gam/__init__.py

```diff
@@ -555,6 +556,23 @@ def _getCfgRowFilter(itemName):
     if not GC_Values[GC_NO_UPDATE_CHECK]:
         doGAMCheckForUpdates()
 
+# domain must be set and customer_id must be set and != my_customer when enable_dasa = true
+    if GC_Values[GC_ENABLE_DASA]:
+        if not GC_Values[GC_DOMAIN]:
+            controlflow.system_error_exit(
+                3,
+                f'Environment variable GA_DOMAIN must be set when enabledasa.txt is present'
```

Comment:
can we show the full path to enabledasa.txt here to help the user know what file to delete if needed?